frontend/src/app/Settings/Users/FacilityAccessTab.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
```

Comment:
This mostly comes from the now deleted `UserDetail` file.
View the [actual diff here](https://www.diffchecker.com/e8M8PU8e/)